go.mod

```diff
@@ -42,10 +43,10 @@ require (
 	github.com/pingcap/goleveldb v0.0.0-20171020122428-b9ff6c35079e
 	github.com/pingcap/kvproto v0.0.0-20190910074005-0e61b6f435c1
 	github.com/pingcap/log v0.0.0-20191012051959-b742a5d432e9
-	github.com/pingcap/parser v0.0.0-20191025082927-f8adf1670b97
+	github.com/pingcap/parser v0.0.0-20191030075336-d8fa9d7074cb
 	github.com/pingcap/pd v1.1.0-beta.0.20190923032047-5c648dc365e0
 	github.com/pingcap/tidb-tools v2.1.3-0.20190321065848-1e8b48f5c168+incompatible
-	github.com/pingcap/tipb v0.0.0-20191022094114-a2e8c3fa634b
+	github.com/pingcap/tipb v0.0.0-20191030045153-07a0962bbc64
```

Comment:
Is it expected to update `tipb` version in this PR?

Reply:
I've reset this change...